src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -78,7 +78,8 @@ class _Durations(NamedTuple):
 class _SimulationEventNames(NamedTuple):
```

Comment:
What is this named tuple used for? It's a bit annoying that we'll need to keep this in sync with the list of simulation events defined in the model spec.

Reply:
I am using in (not in this component) but to define the constants here and then check in components on time step and compare them to the constants in this named tuple. But I do agree it is annoying because I can't import this into the model spec.